models/rate_limits.go

```diff
@@ -23,11 +23,9 @@ type InstanceRateLimits struct {
 // This denotes the rate limits on each individual route.
 type OprishRateLimits struct {
 	// The rate limit information for the `/` route.
-	Info RateLimitConf `json:"info"`
+	Info RateLimitConf `json:"get_instance_info"`
```

Comment:
These seem to have changed https://next.eludevs.pages.dev/reference/todel/OprishRateLimits/